### PR TITLE
Add docs about npm and npm audit

### LINKED ISSUE #22: Document how to find and resolve security vulnerabilities
We currently have a vuln with node-growl but to me it's unclear which of our dependencies actually depend on it?

Can we document how we resolve these vulnerabilities?

### PR DESCRIPTION
**Pull request by megheaiulian**
fixes #22

### CODE REVIEW

File: docs/intro/10-development-environment.rst [hunk 1]
```diff
@@ -72,6 +72,18 @@ Windows:
 
 .. _yarn-setup:
 
+NPM *
+-------
+
+NPM is the package manager for JavaScript and the world’s largest software registry.
+A ``package-lock.json`` file should be synced with ``yarn.lock`` by using a package.json script:
+    ``"sync-npm": "rm -f package-lock.json && npm shrinkwrap && mv npm-shrinkwrap.json package-lock.json"``
```
Comment (nomego) [inline docs/intro/10-development-environment.rst]: The sync-npm command can be read in package.json, I think it's more important to mention why and when to run ``yarn sync-npm``

File: docs/intro/10-development-environment.rst [hunk 2]
```diff
@@ -72,6 +72,18 @@ Windows:
...
+That ``package-lock.json`` file is need for dependency analysis and auditing security issues.
+
+Starting with version 6 ``npm audit`` will list security vulnerabilities found in the currently installed packages.
```
Comment (nomego) [inline docs/intro/10-development-environment.rst]: How do we find the responsible package and fix them?

**Review by nomego — CHANGES_REQUESTED** [review]

File: docs/intro/10-development-environment.rst (hunk 1 above)
Comment (megheaiulian) [reply docs/intro/10-development-environment.rst]: This isn't something that a developer should worry about. We just add it to package.json and call it in `postinstall`. All that the developer should do to have everything working is run `yarn`.

File: docs/intro/10-development-environment.rst (hunk 2 above)
Comment (megheaiulian) [reply docs/intro/10-development-environment.rst]: Simply running `npm audit` will list security issues and how to fix them if possible.

**Comment by megheaiulian** [comment]
@nomego Updated info.

**Pull request merged**